lib/gssapi/spnego/accept_sec_context.c

```diff
@@ -454,7 +462,7 @@ acceptor_complete(OM_uint32 * minor_status,
 	    ret = verify_mechlist_mic(minor_status, ctx, mech_buf, mic);
 	    if (ret) {
 		if (*get_mic)
-		    send_reject (minor_status, output_token);
+		    send_reject (minor_status, GSS_C_NO_BUFFER, output_token);
```

Comment:
Why `GSS_C_NO_BUFFER` in this case?

Reply:
Ah, because we don't have one yet.

Reply:
No, we do have one, but I guess we don't want to complete the inner context.  Good.